packages/botonic-core/src/types/hubtype-service.d.ts

```diff
@@ -9,15 +9,33 @@ interface UnsentInput {
   unsentInput: Input
 }
 
-interface HubtypeServiceArgs {
+export interface HubtypeServiceArgs {
   appId: string
   user: SessionUser
   lastMessageId: string
   lastMessageUpdateDate: string
   onEvent: any
   unsentInputs: UnsentInput[]
+  server: any
```

Comment:
What is `server`? Can be typed with a different type than `any`? 
We should avoid typing attributes as `any`.

Reply:
yes, this is `ServerConfig` type, added 👍